aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -226,13 +229,13 @@ std::pair<const AnnotatedKernel&, const char*> OperatorEntry::computeDispatchTab
   //          A CompositeExplicitAutograd kernel prevents CompositeImplicitAutograd kernel being used for Autograd keys, but it doesn't
   //          cause confusion for AutogradOther. It's pretty straightforward to use Autograd (if available)
   //          in this case.
-  //    (2.3) Use kernel from DispatchKey::Autograd if available
+  //    (2.4) Use kernel from DispatchKey::Autograd if available
   //    The implementation of (2.2) relies on the invariant that for a given backend,
   //    `computeDispatchTableEntryWithDebug()` will be called for that backend's autograd key after the
   //    backend key. See Note [Refresh Runtime Autograd entries in dispatchTable_]
   //  (3) Use fallthrough kernel that are registered as fallback.
   // Alias Key Precedence:
-  //   CompositeExplicitAutograd > CompositeImplicitAutograd > Autograd
+  //   CompositExplicitAutogradWithMutations > CompositeExplicitAutograd > CompositeImplicitAutograd > Autograd
```

Comment:
I figured that the `CompositeExplicitAutogradWithMutations` kernels should get higher precedence, since they belong to a more specific group of dispatch keys than `CompositeExplicitAutograd`

Reply:
I don't follow. I could have renamed these `CompositeExplicitAutograd` and `CompositeExplicitAutogradWithoutMutations` and it seems to me like the latter is more specific!

Reply:
I meant more specific in terms of "the number of dispatch keys in the alias set". `size(CompositeExplicitAutogradWithMutations)` < `size(CompositExplicitAutograd)` < `size(CompositeImplicitAutograd)`.

I guess this also shouldn't matter too much in practice, since we shouldn't really ever have an op that has both a `CompositExplicitAutogradWithMutations` and a `CompositExplicitAutograd` kernel.